diff --git a/node_modules/@changesets/get-github-info/dist/changesets-get-github-info.cjs.js b/node_modules/@changesets/get-github-info/dist/changesets-get-github-info.cjs.js
index a74df59..c063d04 100644
--- a/node_modules/@changesets/get-github-info/dist/changesets-get-github-info.cjs.js
+++ b/node_modules/@changesets/get-github-info/dist/changesets-get-github-info.cjs.js
@@ -175,7 +175,17 @@ const GHDataLoader = new DataLoader__default["default"](async requests => {
     body: JSON.stringify({
       query: makeQuery(repos)
     })
-  }).then(x => x.json());
+  }).then(x => {
+    console.log('going to try x.json()');
+    try {
+      const json = x.json();
+      console.log('Nevermind it worked, oh well! try again!');
+      return json;
+    } catch (e) {
+      console.log('This is the error!');
+      console.log(e);
+    }
+  });
 
   if (data.errors) {
     throw new Error(`An error occurred when fetching data from GitHub\n${JSON.stringify(data.errors, null, 2)}`);
